azext_iot/central/services/device.py

```diff
@@ -9,22 +9,26 @@
 
 from knack.util import CLIError
 from knack.log import get_logger
-from typing import List
 
 from azext_iot.constants import CENTRAL_ENDPOINT
 from azext_iot.central.services import _utility
-from azext_iot.central.models.device import Device
-from azext_iot.central.models.enum import DeviceStatus
+from azext_iot.central import models as central_models
+from azext_iot.central.models.enum import DeviceStatus, ApiVersion
 from azure.cli.core.util import should_disable_connection_verify
 
 logger = get_logger(__name__)
 
-BASE_PATH = "api/preview/devices"
+BASE_PATH = "api/devices"
 
 
 def get_device(
-    cmd, app_id: str, device_id: str, token: str, central_dns_suffix=CENTRAL_ENDPOINT,
-) -> Device:
+    cmd,
+    app_id: str,
+    device_id: str,
+    token: str,
+    central_dns_suffix=CENTRAL_ENDPOINT,
+    api_version=ApiVersion.v1.value,
+):
```

Comment:
You can use the `Union`  type to indicate the function can return different data types like this:

```
from typing import Union

def get_device(...) -> Union[central_models.DevicePreview,central_models.DeviceV1]
```